.gitlab-ci.yml

```diff
@@ -78,7 +78,7 @@ stages:
     - echo -e "section_start:$(date +%s):build_and_test\r\e[0K
       Build and test ${CI_PROJECT_NAME}"
     # Autodetects host-config created by build_tpls
-    - lalloc 1 -W 20 ${LSF_ACCOUNT} scripts/llnl_scripts/build_src.py -v --automation
+    - lalloc 1 -W 25 ${LSF_ACCOUNT} scripts/llnl_scripts/build_src.py -v --automation
```

Comment:
We were apparently too close to the limit. It was occasionally failing due to time limit

Reply:
Any downside to raising it higher -- e.g. 40 mins? to avoid unnecessary timeouts?